mixins/StoreMixin.js

```diff
@@ -76,9 +78,15 @@ var StoreMixin = {
   store_addListeners: function () {
     Object.keys(this.store_listeners).forEach(function (storeID) {
       var listenerDetail = this.store_listeners[storeID];
+      var events = listenerDetail.events;
 
+      // Check that we actually have events to fire events on
+      if (typeof events !== 'object' || !Object.keys(events).length) {
```

Comment:
For some reason, the `process.env.NODE_ENV` is not set here. I think we need to add a webpack plugin: https://github.com/webpack/webpack/issues/868#issue-60380111